megamek/src/megamek/client/ui/swing/tileset/TilesetManager.java

```diff
@@ -659,4 +627,29 @@ public synchronized void reset() {
         mechImages.clear();
         hexTileset.clearAllHexes();
     }    
+    
+    /** Returns the number of available ultralight destroyed bottom decal images. */
+    private int getULightDecalCount() {
+        return getUltraDecalImgCount(FILENAME_SUFFIX_WRECKS_ULTRALIGHT);
+    }
+    
+    /** Returns the number of available ultraheavy destroyed bottom decal images. */
+    private int getUHeavyDecalCount() {
+        return getUltraDecalImgCount(FILENAME_SUFFIX_WRECKS_ASSAULTPLUS);
+    }
+    
+    private int getUltraDecalImgCount(String suffix) {
```

Comment:
What's so Ultra about it?

Reply:
I called it ultra because it looks for the ultralight and ultraheavy decals... or maybe assaultplus and very light... had to use _some_ name

Reply:
Well, anyway, it's probably a better approach than what I was using.